package.json

```diff
@@ -146,6 +146,7 @@
     "eth-url-parser": "^1.0.4",
     "ethers": "^6.11.1",
     "ethers5": "npm:ethers@5.7.2",
+    "eventemitter2": "5.0.1",
```

Comment:
Needed because of wagmi having `@metamask/sdk` as a dep, itself having `eventemitter2` as a dep. This means that instead of consuming the one from hdwallet in KK event handlers e.g these guys
https://github.com/shapeshift/web/blob/542e276ec801963e26e9c5bfbc43b40108707e6f/src/context/WalletProvider/KeepKey/hooks/useKeepKeyEventHandler.ts#L257
we not consume the one from `@metamask/sdk` which has a lightly diff signature and rugs types